homepage/homepage.css

```diff
@@ -0,0 +1,146 @@
+:root{
+    --border-color:rgba(94, 91, 91, 0.493);
+    --footer-color:#242729
+}
+
+.span {
+    color: var(--main-Cta-color);
```

Comment:
please change to proper naming convention(kebab case)

Reply:
okay will be following it from next time